src/test/java/it/kamaladafrica/codicefiscale/Issue30Test.java

```diff
@@ -0,0 +1,29 @@
+package it.kamaladafrica.codicefiscale;
+
+import static org.junit.Assert.assertEquals;
+
+import java.time.LocalDate;
+
+import org.junit.Test;
+
+public class Issue30Test {
+
+	private final Person PERSON = Person.builder().firstname("NLA").lastname("TTV")
+			.birthDate(LocalDate.of(1961, 2, 15)).isFemale(false)
+			.city(City.builder().name("UNIONE SOVIETICA").prov("EUROPA").belfiore("Z135").build()).build();
+
+	@Test
+	public void test_esteri_codes() {
```

Comment:
You should check against CityProvider, like this:
```java
CityProvider provider = CityProvider.ofDefault();
assertEquals("UNIONE SOVIETICA", provider.findByBelfiore("Z135").getName());
assertEquals("Z135", provider.findByName("UNIONE SOVIETICA").getBelfiore());
```
You don't need to build a person as well as CodiceFiscale objects.